api/src/main/java/org/codice/ddf/admin/api/handler/report/ProbeReport.java

```diff
@@ -74,8 +74,18 @@ public ProbeReport probeResults(Map<String, Object> results) {
         return probeResults;
     }
 
+    public <T> T getProbeResult(String key) {
+        if(probeResults.get(key) != null) {
```

Comment:
I looked at this again, I was curious whether typecasting a null would result in a NPE or class casting exception. Turns out neither since null is a valid value for any object (with the exception of some primitives). It makes sense what @coyotesqrl was saying now, the null check is pointless and you can just do `return (T) probeResults.get(key)`